src/test/resources/application-test.properties

```diff
@@ -22,8 +22,8 @@ api.scottish.parliament=https://data.parliament.scot/api/members
 api.ni.assembly=http://data.niassembly.gov.uk/members.asmx/GetAllCurrentMembers
 api.european.parliament=http://www.europarl.europa.eu/meps/en/download/advanced/xml?countryCode=GB
 api.welsh.assembly=https://senedd.assembly.wales/mgwebservice.asmx/GetCouncillorsByWard
-api.country.register=https://country.register.gov.uk/records.json?page-size=5000
-api.territory.register=https://territory.register.gov.uk/records.json?page-size=5000
+api.country.register=http://localhost:8085/countries/local
```

Comment:
We can't hardcode the port number here, as it's variable.

Reply:
How do I fix that?

Reply:
there are a few references to fixed port localhost in this and other properties files - how is this different?

Reply:
I do not know how to fix this. The URL (URI really) is referencing its own Rest Controller, so how do I then reference localhost with the correct port?? do we need some clever config to do this? or should I just reference the file in github like I did in the first solution?

Reply:
I misread this : I have fixed the url with using the server.port variable.

Reply:
Change to use the port variable have been completed.